apps/photos/src/components/SingleInputForm.tsx

```diff
@@ -88,40 +97,86 @@ export default function SingleInputForm(props: SingleInputFormProps) {
             validationSchema={validationSchema}
             validateOnChange={false}
             validateOnBlur={false}>
-            {({ values, errors, handleChange, handleSubmit }) => (
+            {({
+                values,
+                errors,
+                handleChange,
+                handleSubmit,
+                setFieldValue,
+            }) => (
                 <form noValidate onSubmit={handleSubmit}>
                     {props.hiddenPreInput}
-                    <TextField
-                        hiddenLabel={props.hiddenLabel}
-                        variant="filled"
-                        fullWidth
-                        type={showPassword ? 'text' : props.fieldType}
-                        id={props.fieldType}
-                        name={props.fieldType}
-                        {...(props.hiddenLabel
-                            ? { placeholder: props.placeholder }
-                            : { label: props.placeholder })}
-                        value={values.inputValue}
-                        onChange={handleChange('inputValue')}
-                        error={Boolean(errors.inputValue)}
-                        helperText={errors.inputValue}
-                        disabled={loading}
-                        autoFocus={!props.disableAutoFocus}
-                        autoComplete={props.autoComplete}
-                        InputProps={{
-                            endAdornment: props.fieldType === 'password' && (
-                                <ShowHidePassword
-                                    showPassword={showPassword}
-                                    handleClickShowPassword={
-                                        handleClickShowPassword
-                                    }
-                                    handleMouseDownPassword={
-                                        handleMouseDownPassword
+                    {props.optionsList && props.optionsList.length > 0 ? (
```

Comment:
can we just always have the `<Autocomplete> ` wrapper and if we don't pass options it would not render any suggestion dropdown ?

Reply:
Ideally we should not be adding a if else and render two types of components from one component. 

So we should ideally have different component for singleInputForm with auto complete